modules/govuk/manifests/node/s_db_admin.pp

```diff
@@ -34,6 +34,17 @@
   class { '::govuk::apps::signon::db': } ->
   class { '::govuk::apps::whitehall::db': }
 
+  # include all PostgreSQL classes that create databases and users
```

Comment:
The class that configures the postgres client tools and creds should be at the start of this chain, I think.